source/docs/software/wpilib-tools/robot-characterization/analyzing-feedback.rst

```diff
@@ -30,13 +30,13 @@ To specify the correct settings for your PID controller, use the following optio
 - **Controller Period:** This is the execution period of the control loop, in seconds. The default RIO loop rate is 50Hz, corresponding to a period of 0.02s. The onboard controllers on most "smart controllers" run at 1Khz, or a period of 0.001s.
 - **Max Controller Output:** This is the maximum value of the controller output, with respect to the PID calculation. Most controllers calculate outputs with a maximum value of 1, but Talon controllers have a maximum output of 1023.
 - **Time-Normalized Controller:** This specifies whether the PID calculation is normalized to the period of execution, which affects the scaling of the D gain.
-- **Controller Type:** This specifies whether the controller is an onboard RIO loop, or is running on a smart motor controller such as a Talon or a SPARK Max.
+- **Controller Type:** This specifies whether the controller is an onboard RIO loop, or is running on a smart motor controller such as a Talon or a SPARK MAX.
 - **Post-Encoder Gearing:** This specifies the gearing between the encoder and the mechanism itself. This is necessary for control loops that do not allow user-specified unit scaling in their PID computations (e.g. those running on Talons). This will be disabled if not relevant.
 - **Encoder EPR:** This specifies the edges-per-revolution (not cycles per revolution) of the encoder used, which is needed in the same cases as Post-Encoder Gearing.
 - **Has Follower:** Whether there is a motor controller following the controller running the control loop, if the control loop is being run on a peripheral device. This changes the effective loop period.
-- **Follower Update Period:** The rate at which the follower (if present) is updated. By default, this is 100Hz (every 0.01s) for Talons and Sparks, but can be changed.
+- **Follower Update Period:** The rate at which the follower (if present) is updated. By default, this is 100Hz (every 0.01s) for the Talon and the SPARK, but can be changed.
```

Comment:
I think this should specify Talon SRX, Falcon, and Spark Max. Non-CAN talons don't have followers.